pom.xml

```diff
@@ -169,7 +169,7 @@
       <archive>https://groups.google.com/forum/#!forum/checkstyle-announce</archive>
       <subscribe>mailto:checkstyle-announce%2Bsubscribe@googlegroups.com</subscribe>
       <unsubscribe>mailto:checkstyle-announce%2Bunsubscribe@googlegroups.com</unsubscribe>
-      <post>privilege of admins</post>
+      <!-- Only admins are allowed to post. -->
```

Comment:
All links now required to be a valid URL:
https://github.com/apache/maven-project-info-reports-plugin/commit/98db5e5289ca43147a8b4e37d8acffa8ef74e1c6#diff-e1cc4963012830db7457847a37160940fd49a35aadaa1947e4a2cf6384909195R282

Reply:
we simply need to remove whole `<mailingList>` 
We abandoned https://groups.google.com/g/checkstyle-announce a long time ago.

Reply:
done